rules/windows/process_creation/proc_creation_win_sharp_chisel_usage.yml

```diff
@@ -0,0 +1,32 @@
+title: SharpChisel Usage
+id: cf93e05e-d798-4d9e-b522-b0248dc61eaf
+status: experimental
+description: Detects usage of the Sharp Chisel via the commandline arguments
+author: Nasreddine Bencherchali
+references:
+    - https://github.com/shantanu561993/SharpChisel
+    - https://www.sentinelone.com/labs/wading-through-muddy-waters-recent-activity-of-an-iranian-state-sponsored-threat-actor/
+date: 2022/09/05
+logsource:
+    category: process_creation
+    product: windows
+detection:
+    selection_img:
+        Image|endswith: '\SharpChisel.exe'
+    selection_client_server:
+        CommandLine|contains:
+            - 'exe client '
+            - 'exe server '
```

Comment:
Realy need the `exe` ?

Reply:
Just to make sure it's at the starting argument and not in the middle or end. To avoid some potential FP

Reply:
I don't like the perfect string match search but it made sense here